aviary/utils/Fortran_to_Aviary.py

```diff
@@ -39,7 +38,7 @@
 
 
 def create_aviary_deck(fortran_deck: str, legacy_code=None, defaults_deck=None,
-                       out_file=None, force=False):
+                       out_file=None, force=False, verbosity=Verbosity.QUIET):
```

Comment:
I feel like the default ought to be Verbosity.Brief.  There is a certain amount of information that we probably should print out for the user (I think Overrides fall into this category.)  Then, if they want to clear everything out, they can go quiet (and we might even suppress openmado warnings for that one.)

I could also be persuaded that Verbose should be default. I think granularizing our messages between Brief and Verbose will take some thought.  Of course, that leaves Debug for the kitchen sink.

Reply:
hmm, maybe:
debug - everything
verbose - default
brief - only warnings
quiet - nothing

Reply:
or:
debug - everything
verbose - optimizer output
brief - default
quite - nothing

Reply:
I lean towards that second description. However I think we need to let the user know if the optimization was successful or not in that case, even if we don't show them the whole optimizer output

Reply:
I agree with @Kenneth-T-Moore about quiet operation suppressing non-critical warnings. That was kind of the intent for that mode, to suppress literally everything that wasn't a show-stopping bug. That way big parametric studies won't generate pages of terminal output, which really isn't helpful to anyone.